pom.xml

```diff
@@ -29,26 +29,26 @@
 		<java.version>1.8</java.version>
 		<javadoc.opts>-Xdoclint:none</javadoc.opts>
 
-		<spring-cloud-dataflow-ui.version>1.7.0.RELEASE</spring-cloud-dataflow-ui.version>
+		<spring-cloud-dataflow-ui.version>1.7.0.RC1</spring-cloud-dataflow-ui.version>
```

Comment:
Why is this RC1?

Reply:
I guess this was updated earlier and needs to be at `1.7.0.RELEASE`.

Reply:
Merge mistake